src/components/InputText/InputText.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import InputText from './InputText';
+
+const meta = {
+  title: 'Components/Text input',
+  component: InputText,
+  parameters: {
+    layout: 'centered'
+  },
+  tags: ['autodocs'],
+  args: {
+    label: 'Text input'
+  }
+} satisfies Meta<typeof InputText>;
+
+export default meta;
+type Story = StoryObj<typeof InputText>;
+
+const helpText = 'This is an example of help text. It can wrap to two lines, but try not to go longer than three.';
+const errorText = 'This is an example of an error message.';
+
+export const Playground: Story = {};
+
+export const WithPlaceholder: Story = {
+  args: {
+    placeholder: 'Placeholder text'
+  }
+};
```

Comment:
add `WithPlaceholder.storyName = 'With placeholder';` to align with casing guidelines